girder/api/v1/resource.py

```diff
@@ -223,13 +232,17 @@ def _lookUpPath(self, path, user):
                'path starting with either "/user/[user name]", for a user\'s '
                'resources or "/collection/[collection name]", for resources '
                'under a collection.')
+        .param('test',
+               'Specify whether to return null if path doesn\'t exist.',
```

Comment:
`Specify whether to return None instead of throwing an exception when path doesn\'t exist.`

Sorry for being so nit-picky.